test/test_csot.py

```diff
@@ -77,6 +77,7 @@ def test_timeout_nested(self):
     @client_context.require_change_streams
     def test_change_stream_can_resume_after_timeouts(self):
         coll = self.db.test
+        coll.insert_one({})
```

Comment:
This was needed because when I ran this test alone on a fresh cluster it failed with " cannot start $changeStream on non-existent database pymongo_test".

Reply:
NIT: Should we use `create_collection`, or does it not matter?

Reply:
We could but insert_one is easier since it would fail if the collection already exists.